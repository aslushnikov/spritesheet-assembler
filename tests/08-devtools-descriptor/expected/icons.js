// Copyright 2017 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

/**
 * NOTICE: This file was generated by spritesheet_assembler.
 */

UI.Icon.setSpriteSheet({
  size: '50px 50px',
  descriptors: {
    'sprite1': {x: 0, y: 0, width: 20, height: 20},
    'sprite2': {x: -25, y: 0, width: 20, height: 20},
    'sprite3': {x: 0, y: -25, width: 20, height: 20},
    'sprite4': {x: -25, y: -25, width: 20, height: 20},
  }
});