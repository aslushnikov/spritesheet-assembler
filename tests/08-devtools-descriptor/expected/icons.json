{
  "name": "spritesheet",
  "width": 45,
  "height": 45,
  "icons": {
    "sprite1": {"x": 0, "y": 0, "width": 20, "height": 20},
    "sprite2": {"x": -25, "y": 0, "width": 20, "height": 20},
    "sprite3": {"x": 0, "y": -25, "width": 20, "height": 20},
    "sprite4": {"x": -25, "y": -25, "width": 20, "height": 20}
  }
}